tests/Aurora.Shared.Tests/Base/Parent.cs

```diff
@@ -1,11 +1,3 @@
 ﻿namespace Aurora.Shared.Tests.Base;
 
-public class Parent
-{
-    public Parent(int parentProp)
-    {
-        ParentProp = parentProp;
-    }
-
-    public int ParentProp { get; set; }
-}
+public abstract record Parent(int ParentProp);
```

Comment:
Maybe add IdentityVersioned thing ???? this could describe Version and Audit (who and when modified), probably useful, but maybe not

Reply:
This is a test class.